test/e2e/app-dir/segment-cache/basic/segment-cache-basic.test.ts

```diff
@@ -0,0 +1,28 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('segment cache (basic tests)', () => {
+  const { next, isNextDev, isNextDeploy } = nextTestSetup({
+    files: __dirname,
+  })
+
+  if (isNextDev || isNextDeploy) {
+    test('ppr is disabled', () => {})
+    return
+  }
```

Comment:
we can pass `skipDeployment: true` to `nextTestSetup` to avoid creating an unused deployment. `nextTestSetup` will return a `skipped` value that you can use in place of `isNextDeploy` here.

```suggestion
  const { next, isNextDev, isNextDeploy, skipped } = nextTestSetup({
    files: __dirname,
    skipDeployment: true
  })

  if (isNextDev || skipped) {
    test('ppr is disabled', () => {})
    return
  }
```